src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -149,6 +149,35 @@ def acr_build_task_list(cmd,
         cmd.cli_ctx, registry_name, resource_group_name, BUILD_TASKS_NOT_SUPPORTED)
     return client.list(resource_group_name, registry_name)
 
```

Comment:
Two empty lines between methods.

Reply:
Removed.

Reply:
No, not remove, we want 2 empty lines between methods.